### PR TITLE
[Bug Fixes] Fixing Email Preview Gap and Keyboard Navigation

### PR DESCRIPTION
**Pull request by fnormand01**
# Description

This changes in this PR are fixing the following bugs:
- Gap at the top of the email preview list
- Email preview keyboard navigation

## Type of change

- Bug fix (non-breaking change which fixes an issue)

## Screenshot

**Gap Bug**
---

**Before:**

image

**After:**

image

**Keyboard Navigation Bug**
---

**Mouse Over and Keyboard Focus:**

image

# Testing

Manual steps to reproduce this functionality:

1.  Start the eMIB Sample Test.
2.  Complete the _pre-test_ process.
3.  Open the _Inbox_ tab.
4.  Make sure the the gap bug is fixed.
5.  Also, make sure that you are able to navigate in the email preview list using the keyboard and everything look good (mouse over and focus).

# Checklist

Applicable for all code changes.

- [ ] ~~I have commented my code, particularly in hard-to-understand areas~~
- [ ] ~~I have made corresponding changes to the documentation~~
- [x] My changes generate no new compiler warnings
- [ ] ~~My changes generate no new accessibility errors and/or warnings~~
- [ ] ~~I have added tests that prove my fix is effective or that my feature works~~
- [x] I have researched WCAG2.1 accessibility standards and met them in this PR (can be navigated using a keyboard)
- [x] My changes look good on IE 10+, Firefox, and Chrome

### CODE REVIEW

**Comment by caleybrock** [comment]
Let's checkin with @joeyhua when she's back on these active states.

**Comment by joeyhua** [comment]
I think have a hover indication is good, but might be more consistent and stronger with a bg colour change instead of the border. This is an assumption though, we could test the border.

**Comment by joeyhua** [comment]
@fnormand01 Actually, can we have the background colour for the hover state be #C6ECEF? No border

**Comment by fnormand01** [comment]
Made changes suggested by @joeyhua in commit #https://github.com/code-for-canada/project-thundercat/pull/201/commits/a9b9894ec98c5aea61d09d7550147b927be828ab.

File: frontend/src/css/inbox.css [hunk 1]
```diff
@@ -19,3 +19,11 @@ styles that cannot be added inside a const
   -ms-grid-column: 2;
   grid-column: 2;
 }
+
+.email-preview-button:focus {
+  background-color: #c6ecef !important;
```
Comment (caleybrock) [inline frontend/src/css/inbox.css]: Anytime you use important, can you add a comment explaining why it's necessary?

Comment (fnormand01) [reply frontend/src/css/inbox.css]: Comment added in commit #https://github.com/code-for-canada/project-thundercat/pull/201/commits/f4f8a14ad766d538b60b9b444f2124f6a1abd87b

**Pull request merged**